priv/templates/script.upload.eex

```diff
@@ -125,4 +149,10 @@ if [ "$(uname -s)" = "Darwin" ]; then
     fi
 fi
 
-cat $FILENAME | ssh -s $SSH_OPTIONS $DESTINATION nerves_firmware_ssh2
+# Check if the system is still running the old NervesFirmwareSSH which
+# is listening on port 8989
+if (echo >/dev/tcp/$DESTINATION/8989) &>/dev/null; then
+    printf "fwup:$FILESIZE,reboot\n" | cat - $FILENAME | ssh -s -p 8989 $SSH_OPTIONS $DESTINATION nerves_firmware_ssh
+else
+    cat $FILENAME | ssh -s $SSH_OPTIONS $DESTINATION nerves_firmware_ssh
+fi
```

Comment:
It looks like your editor is trimming the last line's newline character. That generates a warning when viewing the git diffs. Looks like github renders it as a weird circle line thing. Could you turn off whatever is doing that?

Reply:
Weird. My editor says it should be adding newlines...not removing 🤔 I'll see wha I can do